Neos.ContentRepository.NodeMigration/src/NodeMigrationService.php

```diff
@@ -54,8 +53,7 @@ public function __construct(
         private readonly ContentRepository $contentRepository,
         private readonly FiltersFactory $filterFactory,
         private readonly TransformationsFactory $transformationFactory
-    )
-    {
+    ) {
```

Comment:
Really? Is this a special case?

We use 
```php
public function __construct(array $foo, string $bar)
{
}
```
elsewhere. And even for multi-line arguments my tooling would create:
```php
public function __construct(
    array  $foo,
    string $bar
)
{
}
```

Reply:
I'm fine with both. I even thought this is PSR, but seems just to be the Symfony standard. I'll change that.

Reply:
PSR-12 doesn't mention constructors explicitly, but in general for functions/methods it's

```
public function anotherFunction(
    string $foo,
    string $bar,
    int $baz
): string {
    return 'foo';
}
```

See https://www.php-fig.org/psr/psr-12/

Reply:
So, it is PSR-12 than:
> When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them.

https://www.php-fig.org/psr/psr-12/#45-method-and-function-arguments

WDYT @kdambekalns ?

Reply:
```
    protected function executeSubMigrationAndBlock(
        array $migrationDescription,
        ContentStreamId $contentStreamForReading,
        ContentStreamId $contentStreamForWriting
    ): void
    {
```

existing 9.0 example, so I guses we do it nicely consistent with { on a new line, which I like too :)

Reply:
Hmm, there are also a lot of counterexamples 🤷‍♂️ 

E.g: https://github.com/neos/neos-development-collection/blob/4836dddc0d9ef3f0476c0b3f89b8997eb53a218b/Neos.ContentGraph.DoctrineDbalAdapter/src/Domain/Projection/HierarchyRelation.php#L37-L38

Reply:
YIKES, INCOSISTENCY HALP 😆  I guess we need a vote somehow?

Reply:
I'd vote for a standard, rather than single rules. As this is somehow hard to communicate to contributers if we have a list of "exceptions" from a choosen standard.

Reply:
true, although I find in this case the standard declares annoying single rules. opening brace on a new line would be a good basic rule, but no it depends on if the arguments are multi line or not.

But yes, PSR-12 it is :)

Reply:
i vote for 

```
 ) { 
 } 
 ```

and thats what we used so far in neos 8.x - and it looks pretty.

Reply:
it seems like we have examples for everything in our code base -> see `{` on same line: https://github.com/neos/neos-development-collection/blob/373d253252d1a8b0f0229d86bb51d59788b4e842/Neos.ContentRepositoryRegistry/Classes/ContentRepositoryRegistry.php#L116

but this pattern was only used 13 times (all new with 9.0 - i think sebastians code style)

```
public function foo(
   ...$args
)
{
}
```

while there are currently already 100+ usages of the way i prefer:

```
public function foo(
   ...$args
) {
}
```

so i rather like to adjust the 13 cases to the norm than the other way around ;)

Reply:
so id say we merge this if no one opposes?